types/culori/tsconfig.json

```
@@ -0,0 +1,22 @@
+{
+    "compilerOptions": {
+        "noEmit": true,
+        "lib": ["es6"],
+        "module": "commonjs",
+        "baseUrl": "../",
+        "typeRoots": ["../"],
+        "types": [],
+        "target": "es6",
+        "strict": true,
+        "forceConsistentCasingInFileNames": true,
+        "exactOptionalPropertyTypes": true,
+        "noUncheckedIndexedAccess": true,
+        "noUnusedLocals": false,
+        "noUnusedParameters": true
```

```suggestion
        "module": "commonjs",
        "lib": [
            "es6"
        ],
        "noImplicitAny": true,
        "noImplicitThis": true,
        "strictFunctionTypes": true,
        "strictNullChecks": true,
        "baseUrl": "../",
        "typeRoots": [
            "../"
        ],
        "types": [],
        "noEmit": true,
        "forceConsistentCasingInFileNames": true
```
those are non-standard settings, this is why warning label was added. Why do you want those different settings? thx!

---

I want to make checks of TS compiler as strict as possible. All this compilerOptions I copied from https://github.com/tsconfig/bases/blob/main/bases/node18-strictest.combined.json

---

@peterblazejewicz 

I checked that most of the options I added don't really add any value. But I can't decide whether `exactOptionalPropertyTypes` is useful or not. What do you think about it?